recipes/xsd/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xsd" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.codesynthesis.com/download/{{ name }}/4.0/{{ name }}-{{ version }}+dep.tar.bz2
+  sha256: {{ sha256 }}
+
+  patches:
+    - osx-build.patch
+    # see http://codesynthesis.com/pipermail/xsd-users/2015-June/004638.html
+    - osx-test.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+    - xerces-c 3.1.*
+    - python
+  run:
+    - pkg-config
```

Comment:
The tests use pkg-config. Since i provided the pkg-config file myself I wanted to test if it works. xsd generates source code, the tests make sure this code compiles.